### ISSUE TITLE
Promote blog : Notify email.

### ISSUE THREAD

**Issue by coa0329** [coa0329/BhaangWeb#77]
![163](https://user-images.githubusercontent.com/7914690/30962714-149ec584-a475-11e7-84ff-648368b8b110.JPG)

![164](https://user-images.githubusercontent.com/7914690/30962765-46022508-a475-11e7-9b4e-8fc3a4141164.JPG)

Should "Notify me at" show the email the user select in from ?

**Comment by daver78** [comment]
@coa0329 Hmmm ... good clarification.  Let me ask you and @paagla123: Which address do we use as the `TO:` address in the lead alerts?  

Are you changing it each time to match the `FROM` address of the corresponding email?

**Comment by paagla123** [comment]
@daver78 right now the MAML template sends it to the sender. If we want to change that, we will need to open up another content variable. Thx

**Comment by paagla123** [comment]
@daver78 we should discuss this.

**Comment by coa0329** [comment]
@daver78 @paagla123 

What is the solution ?

**Comment by paagla123** [comment]
@daver78 MAML part is easy. Question is - where in the UI should we put this?

**Comment by paagla123** [comment]
@daver78 this is being moved to next MileStone. I believe this is now covered in the new gatekeeper functionality

**Comment by daver78** [comment]
@paagla123 Are you asking if it is included in the `revamped_schedule` feature flag?

If so, then NO it's not there right now -- but we can add it.  Please confirm which direction you advise.

**Comment by paagla123** [comment]
@daver78 yes - we should add a provision to input the  name and address of the person/group that received the lead email - may be in settings of email

**Comment by paagla123** [comment]
@daver78 can we discuss where we want to have it?

**Comment by daver78** [comment]
@paagla123 Yes let's discuss today.  I will likely work on this tomorrow.

**Comment by daver78** [comment]
Per discussion with @paagla123, I will add a way to Step 3 where you can select from a list of Users you have added to your account.  This means we need to have a way to add Users in Settings.

**Comment by daver78** [comment]
@paagla123 Is @coa0329 assigned to this?  Please add the right person.

---
There are two parts to this:

(a) This part in the settings:

![image](https://user-images.githubusercontent.com/7561473/31740209-7aaca824-b405-11e7-8ab6-d7d6170d11ae.png)

(b) This part in the Blueprint:

![image](https://user-images.githubusercontent.com/7561473/31740274-aeaf3c54-b405-11e7-85e3-c83e171ba386.png)

For the part in the Blueprint:

Controller is `editPromoteBlog.js` around line 39.  You’ll see my mock:

![image](https://user-images.githubusercontent.com/7561473/31740214-80e575b8-b405-11e7-9813-e232206a502e.png)


HTML is `editPromoteBlog_step4_revamped_schedule.php`.

`create-option-text `doesn’t work yet.  I think we have a conflict with JS somewhere

**Comment by coa0329** [comment]
@paagla123 : fixed on stage.

You can Add / Edit and Delete user in settings [new layout] / Users.

I will do the input validation later.

=======

From in Blog and Ebook will get data from settings.

**Comment by daver78** [comment]
@coa0329 Good job!  I will take a look today and @paagla123 will have comments too.

By the way, @coa0329 -- please provide me feedback on how to make the UI easier for you to implement.  For example, was it helpful that I added the items to `$scope` with dummy array?

**Comment by coa0329** [comment]
@daver78 
I didn't work on (b) part yet but they can help.

**Comment by coa0329** [comment]
@paagla123 

part (b) : fixed on stage. I bring save button back. You need to click it to save data to couchdb.

in #140 @daver78 hide publish button. So we cannot publish the revamp version now.

![021](https://user-images.githubusercontent.com/7914690/31816296-ee1bb9d4-b5b9-11e7-9306-ca247d7481ba.JPG)

**Comment by paagla123** [comment]
@coa0329 I will test it at length, but there is one fundamental problem. In the new UI, when we you come to the notifications tab for the first time (for a new campaign) and click on "Open my Emails", the whole page resets and it goes back to the old format. Please fix this - I will test the rest

**Comment by coa0329** [comment]
@jiewsone will check why it goes back to the old format.

**Comment by paagla123** [comment]
@coa0329 is this completed otherwise?

**Comment by jiewsone** [comment]
@paagla123 Sorry, I forgot to update you. I already fix on stage.

**Comment by paagla123** [comment]
Copying @jiewsone - ready to move forward you know. Tx

**Comment by coa0329** [comment]
@daver78 @paagla123 

![036](https://user-images.githubusercontent.com/7914690/32269832-29603a2c-bf26-11e7-919f-7330039a5ce5.JPG)


We can add users in setting. Can we remove "Add this email" in each sendto list ?

**Comment by daver78** [comment]
@coa0329 No, I would prefer not to remove the "Add this email".  Why do you think we should remove it?  I believe the use-case will be fairly common for the User.
